src/sync/streaming/pushManager.ts

```diff
@@ -221,7 +221,17 @@ export function pushManagerFactory(
   /** Functions related to synchronization (Queues and Workers in the spec) */
 
   pushEmitter.on(SPLIT_KILL, splitsUpdateWorker.killSplit);
-  pushEmitter.on(SPLIT_UPDATE, splitsUpdateWorker.put);
+  pushEmitter.on(SPLIT_UPDATE, (parsedData) => {
+    if (parsedData.d !== undefined) {
```

Comment:
`if (parsedData.d)` is shorter and more robust, in case a `null` or empty string is provided